virttest/qemu_storage.py

```diff
@@ -269,6 +269,16 @@ def _get_image_meta(image, params, root_dir):
             meta['file'].update({
                 k: v[1] for k, v in six.iteritems(mapping) if v[0]
             })
+    filters = params.get("image_filters", "").split()
+    # Stack them reversed, so the first filter on the list
+    # will be on top of the rest of filters
+    for img_filter in filters[::-1]:
+        tmp_meta = collections.OrderedDict()
+        tmp_meta["file"] = meta
+        meta = tmp_meta
+        meta["driver"] = img_filter
```

Comment:
remove this line, let filter function itself set its filter name. e.g. copy-on-read filter, we cannot define a _copy-on-read() function, we can assign image filter name as 'cor', define the function as _cor, and let _cor() set {'driver': 'copy-on-read'}